packages/google-closure-compiler/lib/gulp/index.js

```diff
@@ -30,7 +30,13 @@
 
 class CustomError extends Error {
   constructor(plugin, message) {
-    super(`${plugin}: ${message}`);
+    if (message instanceof Error) {
+      super(`Error in ${plugin}`);
+      this.original = message
+      this.stack = `${this.stack.split('\n').slice(0,2).join('\n')}\n${this.stack}`;
```

Comment:
This is a bit of a brain bender, could you add a comment explaining what you're doing to the stack here?